driver-core/src/main/com/mongodb/client/model/expressions/Expressions.java

```diff
@@ -0,0 +1,226 @@
+package com.mongodb.client.model.expressions;
+
+import org.bson.BsonBoolean;
+import org.bson.BsonDateTime;
+import org.bson.BsonDecimal128;
+import org.bson.BsonDocument;
+import org.bson.BsonDocumentWriter;
+import org.bson.BsonDouble;
+import org.bson.BsonInt32;
+import org.bson.BsonInt64;
+import org.bson.BsonString;
+import org.bson.BsonValue;
+import org.bson.codecs.Codec;
+import org.bson.codecs.EncoderContext;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.types.Decimal128;
+
+import java.util.Date;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+
+/**
+ * Builders for aggregate expressions.
+ * 
+ * @mongodb.driver.manual reference/operator/aggregation/
+ */
+public final class Expressions {
```

Comment:
This is just a smattering of expressions.  The idea would be to add all 120 in here.  I considered breaking them out by category, as they are done in the manual, but it seems like keeping them all together helps in discovery as you only need a single static import of `Expressions.*`